client/lib/application/src/approval-form/approval-form.jsx

```diff
@@ -133,6 +133,11 @@ class ApprovalForm extends React.Component {
         const {applicationId, versionId, application, approvalTypes, approvals, editable, userInfos} = this.props,
         LINK_PARAMS = {applicationId, versionId};
         return <div className='approvalForm'>
+                    <div style={{backgroundColor: '#f08532'}}>
```

Comment:
The same component is copied 3 times. Can we consider separate component creation?